src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java

```diff
@@ -65,23 +72,43 @@
                 "ApplyToVary",
                 "ApplyToDischargePeriodicalPaymentOrder"
         ));
-        ALLOWED_VALUES_PER_FIELD.put(APPLYING_FOR_CONSENT_ORDER, asList("Yes"));
+        ALLOWED_VALUES_PER_FIELD.put(APPLYING_FOR_CONSENT_ORDER, Collections.singletonList("Yes"));
```

Comment:
so many bytes wasted!

Reply:
🤷‍♂ IIRC it's actually identical amount of bytes used, just Collections.singletonList is a superrrr small amount more performant -  I'll revert it though sure

Reply:
:-D I meant source code bytes - thank you though - it's about source code reading comfort